src/base/Roles/AccountantWithRateProviders.sol

```diff
@@ -44,6 +44,17 @@ contract AccountantWithRateProviders is Auth, IRateProvider {
         uint16 managementFee;
     }
 
+    /**
+     * @notice Lending specific state
```

Comment:
The comment seems wrong because it doesn't mention lendingRate and protocolFeeRate.